recipes/gnucobol/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "gnucobol" %}
+{% set version = "3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.xz
+  sha256: 3bb48af46ced4779facf41fdc2ee60e4ccb86eaa99d010b36685315df39c2ee2
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - pkg-config
+    - make
+  host:
+    - gmp
+    - json-c
+    - libxml2
+    - ncurses
+    - libdb
+  run:
+    - gmp
+    - json-c
+    - libxml2
+    - ncurses
+    - libdb
+
+test:
+  commands:
+    - cobc --version
```

Comment:
As this is a compiler, could you also add a test that it produces working binaries?